collector/receiver/otelarrowreceiver/README.md

```diff
@@ -1,89 +1,108 @@
-# OTLP Receiver
+# OTel-Arrow Receiver
 
-| Status                   |                       |
-| ------------------------ | --------------------- |
-| Stability                | traces [stable]       |
-|                          | metrics [stable]      |
-|                          | logs [beta]           |
-| Supported pipeline types | traces, metrics, logs |
-| Distributions            | [core], [contrib]     |
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics, traces, logs   |
+| Distributions | [contrib] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fotelarrow%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fotelarrow) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fotelarrow%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fotelarrow) |
 
-Receives data via gRPC or HTTP using [OTLP](
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+Receives telemetry data using the
+[OTel-Arrow](https://github.com/open-telemetry/otel-arrow) protocol
+via gRPC and standard [OTLP](
 https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md)
-format.
+protocol via gRPC or HTTP.
```

Comment:
Hmm can this line be deleted or combined with L16? i.e. L16 seems to cover this (except HTTP part)

Reply:
Done. See also, I've removed optionality.